mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -3826,6 +3828,242 @@
 										}
 									},
 									"response": []
+								},
+								{
+									"name": "list of packages with valid providerId",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"id": "47d2cb7d-710c-44f8-9770-0921df8bb19d",
+												"exec": [
+													"pm.test(\"success test\", function() {",
+													"    pm.response.to.be.json;",
+													"});",
+													"",
+													"let response = pm.response.json();",
+													"",
+													"//Check that status is 200",
+													"pm.test(\"Status is 200\", function () {",
+													"    pm.response.to.have.status(200);",
+													"});",
+													"",
+													"pm.test(\"Validate schema\", function () {",
+													"    pm.expect(tv4.validate(response, JSON.parse(pm.environment.get(\"schema_packageCollection\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
+													"});",
+													"",
+													"//Test that id matches what was provided in query",
+													"pm.test('id matches as provided in query', function(){",
+													"    pm.expect(response.data[0].id.split('-')[0]).eq(pm.environment.get(\"provider-with-tags-id\"));",
+													"});    ",
+													"",
+													"//Test that type is packages",
+													"pm.test('type is packages', function(){",
+													"    pm.expect(response.data[0].type).eq('packages');",
+													"});",
+													"",
+													"//Test that particular package has a correct tag list",
+													"pm.test('type is packages', function(){",
```

Comment:
test should be renamed -> type is packages is name of the previous test

Reply:
Thanks @sduvvuri-ebsco. I will fix it.